installers/metrics/ansible/roles/pgo-metrics/defaults/main.yml

```diff
@@ -9,6 +9,19 @@ metrics_image_pull_secret: ""
 metrics_image_pull_secret_manifest: ""
 pgmonitor_version: "v4.2"
 
+alertmanager_configmap: "alertmanager-config"
+alertmanager_rules_configmap: "alertmanager-rules-config"
+alertmanager_custom_configmap: ""
+alertmanager_custom_rules_configmap: ""
+alertmanager_install: "true"
+alertmanager_image_prefix: "prom"
+alertmanager_image_name: "alertmanager"
+alertmanager_image_tag: "v0.21.0"
+alertmanager_log_level: "info"
+alertmanager_port: "9093"
```

Comment:
should this be an integer?

Reply:
I don't see reason why it couldn't be, after a quick test of updating this port to an integer the deployment and service deployed fine. I did follow how we have it defined for the other ports in both ansible installers so it would probably need to be updated in other places too.